src/index.njk

```diff
@@ -18,10 +18,14 @@ eleventyNavigation:
     </h2>
   </header>
 
-  <p class="hero__desc">
-    We love macOS, but we’re not a fan of the ever-closing hardware and ecosystem. So we are creating ravynOS&nbsp;&mdash;&nbsp;an OS aimed to provide the finesse of macOS with the freedom of FreeBSD.
+  <p align="justify">
+  ravynOS (<b>formerly called airyxOS</b>) is a FreeBSD-based UNIX distribution that aims to be compatible with macOS software, along with the benefits of being free and opensource. Since ravynOS is opensource, this means that ravynOS has no hardware restrictions that Apple enforces for their desktop operating system, thus making it possible to run macOS-built software on various kinds of hardware, and architectures!! 🍻 
```

Comment:
Let's keep the original text and CSS style ("hero__desc") here as well, please. You could add this text after the original text if you want: "ravynOS (previously called airyxOS) is an open-source operating system based on FreeBSD, CMU Mach, and Apple open-source code that aims to be compatible with macOS applications and has no hardware restrictions." It might need to be a separate element to look good.

Reply:
I see, will next commit will be pushed, keeping in mind to what you've requested :slightly_smiling_face: